addon/templates/components/challenges/containter.hbs

```diff
@@ -0,0 +1,10 @@
+<div class="timeline_containter">
+    {{#each model as |milestone|}}
+    {{/each}}
+
+    {{component 'bepstore-goals@challenges/milestone' title="V0.1" showIssues=true openMilestone=openMilestone showMilestone=(action 'showMilestone')}}
+    {{component 'bepstore-goals@challenges/milestone' title="V0.2" openMilestone=openMilestone showMilestone=(action 'showMilestone')}}
```

Comment:
It enables the child to use the function of the parent, 
In this case it is a function which is changing a variable that is observed by all parents